.github/workflows/ci.yaml

```diff
@@ -16,15 +12,11 @@ jobs:
     strategy:
       matrix:
         php-version:
-          - "5.3"
-          - "5.4"
-          - "5.5"
-          - "5.6"
-          - "7.0"
-          - "7.1"
-          - "7.2"
-          - "7.3"
-          - "7.4"
+          - "8.0"
+          - "8.1"
+          - "8.2"
+          - "8.3"
+          - "8.4"
```

Comment:
With 3 years of no updates in this repository and the amount of work to invest to upgrade to a newer PHP version I suggest to cut support for all versions below 8. They are all EOL anyway.